src/simulation/LoadGenerator.cpp

```diff
@@ -515,20 +639,24 @@ LoadGenerator::updateMinBalance()
 
 std::vector<Operation>
 LoadGenerator::createAccounts(uint64_t start, uint64_t count,
-                              uint32_t ledgerNum)
+                              uint32_t ledgerNum, bool initialAccounts)
 {
     vector<Operation> ops;
     SequenceNumber sn = static_cast<SequenceNumber>(ledgerNum) << 32;
+    auto balance = initialAccounts ? mMinBalance * 10000000 : mMinBalance * 100;
     for (uint64_t i = start; i < start + count; i++)
     {
         auto name = "TestAccount-" + to_string(i);
         auto account = TestAccount{mApp, txtest::getAccount(name.c_str()), sn};
-        ops.push_back(
-            txtest::createAccount(account.getPublicKey(), mMinBalance * 100));
+        ops.push_back(txtest::createAccount(account.getPublicKey(), balance));
 
         // Cache newly created account
-        mAccounts.insert(std::pair<uint64_t, TestAccountPtr>(
-            i, make_shared<TestAccount>(account)));
+        auto acc = make_shared<TestAccount>(account);
+        mAccounts.insert(std::pair(i, acc));
```

Comment:
nit: `.emplace(i, acc)` (same below)

Reply:
sure, this was copy pasta from the code in master